sdk/storage/azure-storage-blob-changefeed/setup.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+
+import os
+import re
+
+from setuptools import setup, find_packages
+
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-storage-blob-changefeed"
+NAMESPACE_NAME = "azure.storage.blobchangefeed"
```

Comment:
I feel like the namespace should also be "azure.storage.blob.changefeed"
Is there a reason that it isn't?
I'll discuss it with Johan as well.

Reply:
We need to declare another namespace package if we want to extend this namespace azure.storage.blob 
This is one of the reasons that we are using azure.storage.filedatalake instead of azure.storage.file.datalake for now.

Reply:
In this case we shouldn't need a namespace package because we have a dependency on blobs which _is_ the namespace.

Reply:
We are discussing this in another open comment, so I will resolve this comment!